utils/hooks/useSendQuoteRequest.ts

```diff
@@ -77,7 +75,7 @@ export const useSendQuoteRequest = (): UseSendQuoteRequestReturn => {
   return {
     register,
     handleSubmit: handleSubmit,
-    errors,
+    formState: { errors },
```

Comment:
We can't return `errors` here, because `errors` prop doesn't exist on `UseSendQuoteRequestReturn`; but we can return `formState` and have access to `errors` object through it